packages/core/src/Banner/Banner.spec.js

```diff
@@ -2,6 +2,10 @@ import React from 'react'
 import { Banner, Text, Heading, theme } from '..'
 
 describe('Banner', () => {
+  const consoleError = console.error
+  console.error = jest.fn()
```

Comment:
Would you think the code here can be moved inside the `afterAll` function at L7?

Reply:
It has to be mocked before the tests run otherwise the console errors will be shown as output while the tests are running and `Banner` still needs `bg` to set its colours correctly.